src/components/DropDownButton/index.tsx

```diff
@@ -68,17 +68,23 @@ const Item = styled.div<{ border: boolean }>`
 	border-top: ${props =>
 		props.border ? `1px solid ${props.theme.colors.gray.main}` : '0'};
 	border-radius: ${props => px(props.theme.radius)};
+	background: white;
```

Comment:
Should this be a theme color?

Reply:
@grahammcculloch Above here on line 41 the background is also set directly. shall I also adjust that one?

Reply:
@StefKors ahh - having taken the time to look through the Rendition code-base now I see that 'white' is used all over the place! So this would be part of a larger clean-up/refactor problem and I would ignore my comment above for this PR!